app/src/main/java/org/oppia/app/profileprogress/ProfileProgressHeaderViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.app.profileprogress
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.ObservableField
+import org.oppia.app.home.RouteToRecentlyPlayedListener
+import org.oppia.app.model.Profile
+
+/** Header [ViewModel] for the recycler view in [ProfileProgressFragment]. */
+class ProfileProgressHeaderViewModel(activity: AppCompatActivity) : ProfileProgressItemViewModel() {
+  // TODO(#777): Edit image and shown full screen image popup.
```

Comment:
Please check this todo. It is not clear.

Reply:
`// TODO(#777): Click on ImageView to show a popup which can either edit the image or view image in full screen.`